launch/tier4_simulator_launch/launch/simulator.launch.xml

```diff
@@ -24,6 +24,13 @@
       <arg name="use_object_recognition" value="$(var perception/enable_object_recognition)"/>
       <arg name="visible_range" value="$(var sensing/visible_range)"/>
     </include>
+
+    <!-- replay points to compare map filtered -->
+    <node pkg="topic_tools" exec="relay" name="dummy_perception_to_compare_map_points" output="log">
```

Comment:
@yukkysaito 
I added this because this feature needs compare map filtered points in planning simulation.
is this okay? or are there any other appropriate ways? :thinking:

Reply:
@yukkysaito 
I fixed following items in https://github.com/autowarefoundation/autoware.universe/pull/752/commits/840f1d75f301c07042846e53c9f76147c5182a5e and https://github.com/autowarefoundation/autoware.universe/pull/752/commits/d3c6114cc83cd281cc8fcee272bf31079f92cfbe.
Could you check this change?
- implement dynamic obstacle class as interface class
- select whether to run compare map from launch argument
- subscribe compare map points in this module
- remove relay for compare map points from simulator.launch.xml